### ISSUE TITLE
kpi_without_optimum vs original kpi

### ISSUE THREAD

**Issue by virithavanama** [google/lightweight_mmm#63]
kpi_without_optimum is not matching with the target conversions of the original data

**Comment by pabloduque0** [comment]
Hello @virithavanama !
`kpi_without_optimum` runs a prediction based on historical average media data. If you would like to compare against a different baseline you can always do so.

Can you further explain what you would expect it to match against?

**Comment by virithavanama** [comment]
If the total historical conversions is 48000, in the 'plot_pre_post_budget_allocation_comparison' plot preoptimized predicted target(kpi_without_optimum) is showing as 16000

**Comment by pabloduque0** [comment]
That is possible if your total historic data is eg. 30 weeks and you are optimizing for 10 weeks. Could that be your case? (values are relative not absolute in my example)

**Comment by virithavanama** [comment]
No, we are using complete data for optimizing as well

**Comment by virithavanama** [comment]
n_time_periods = 52
prices = jnp.ones(mmm.n_media_channels)
budget = jnp.sum(jnp.dot(prices, ch_mean))* n_time_periods

solution, kpi_without_optim, previous_budget_allocation = optimize_media.find_optimal_budgets(
    n_time_periods=n_time_periods,
    media_mix_model=mmm,
    #extra_features=extra_features_test,
    budget=budget,
    prices=prices,
    media_scaler=media_scaler,
    target_scaler=target_scaler,
    bounds_lower_pct=0.75,
    bounds_upper_pct=jnp.array([1.5,1.5,1.5,1.5,1.5,1.44,1.5,1.5,1.5]),
    seed=SEED)

**Comment by pabloduque0** [comment]
Thanks! Let me investigate this one and get back to you.

**Comment by pabloduque0** [comment]
I am not able to recreate this great discrepancies with mock data.

What is the output when passing all the extra features as well to the optimisation?

**Comment by virithavanama** [comment]
What if you want to predict the future and then we will not have the extra features right?

**Comment by pabloduque0** [comment]
Depends what those extra features are, for some (holidays, promotions, ...) you might know.  For others no, you will have to estimate them or set them to zero.

However if you extra features have a high positive impact and you want to be optimising for the future without them, output will not match historic absolute values. But I would not focus too much on the absolute value of the optimisation, in this case you know you are missing certain part of the value due to those missing extra features.

**Comment by pabloduque0** [comment]
Discussion on #64 might be helpful as it is related to a very similar topic.
